pkgs/development/tools/xcpretty/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv rec {
```

Comment:
This should probably be `bundlerApp`?. See [the manual](https://nixos.org/nixpkgs/manual/#sec-language-ruby) for the difference.

Reply:
That makes sense.  I changed it to accept a `passthru` argument and to make the `exes` argument optional, defaulting to the `pname`.